packages/calcite-components/src/components/pagination/pagination.scss

```diff
@@ -1,11 +1,26 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-pagination-spacing: Specifies the amount of left and right margin spacing between each item.
```

Comment:
cc @SkyeSeitz / @ashetland - What is the reason for allowing this gap to be customized? Seems like even for "low count" pagination, we'd want to maintain the spacing.

Similar q for "-size" - maybe not needed...

Reply:
+1